ipatests/test_integration/test_commands.py

```diff
@@ -1136,3 +1136,35 @@ def test_cacert_manage(self):
             raiseonerr=False
         )
         assert result.returncode == 0
+
+    def test_ipa_adtrust_install_with_locale_issue8066(self):
+        """
+        This test checks that ipa-adtrust-install command runs successfully
+        on a system with locale en_IN.UTF-8 without displaying error below
+        'IndexError: list index out of range'
+        This is a testcase for Pagure issue
+        https://pagure.io/freeipa/issue/8066
+        """
+        # Set locale to en_IN.UTF-8 in .bash_profile to avoid reboot
```

Comment:
The comment needs to be updated (.bashrc instead of .bash_profile)

Reply:
comment has been updated.